apisix/plugins/authz-casdoor.lua

```diff
@@ -60,7 +61,7 @@ local function fetch_access_token(code, conf)
             ["Content-Type"] = "application/x-www-form-urlencoded"
         }
     })
-
+    connection_util.close_http_connection(client)
```

Comment:
`request_uri` will close(sock) before returning, so this line is not necessary? The same is true elsewhere of the PR.

![image](https://github.com/user-attachments/assets/8a084a61-a294-4d8f-8d7e-a2993c853d02)

Reply:
+1 will remove it.